src/components/Button.js

```diff
@@ -46,26 +59,47 @@ export const Button = ({ disabled, invert, notFullWidth, onPress, title }) => {
       titleStyle={[
         styles.titleStyle,
         invert && styles.titleStyleInvert,
-        needLandscapeStyle && styles.titleStyleLandscape
+        needLandscapeStyle && styles.titleStyleLandscape,
+        (extraLarge || large || medium) && styles.spatialButtonTitle,
```

Comment:
All 3 different button sizes have the same text style. So I added a single text style for 3 different button sizes. I can update the name to make it clearer. Do you have any suggestions for that?

Reply:
yes, like `small && styles.smallButtonTitle` all its own, to be able to change all of those separately depending on requirements.
i guess spatial isn't the word you wanted to use? did you mean special?

Reply:
I added new styles for different text styles and left the styles empty. Since all styles are the same, I added the necessary styles to `titleStyle`. 

Yes special would be more correct.

https://github.com/smart-village-solutions/smart-village-app-app/pull/909/commits/790533a892a73135a641652cecc7dd521444636d

Reply:
i cannot follow this decision. if there are no styles to apply, we do not need any code for that.
with changing `titleStyle` alle buttons will look different after merge? please compare the implicit values with the one you added with the commit. do they differ?

Reply:
But this time we need to consider that now all buttons will appear in this style as standard. I don't know if this is the right way to do it.

|before|after|
|--|--|
![Simulator Screenshot - iPhone 14 Pro - 2023-09-18 at 12 33 18](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/e2301670-eb69-4400-9a22-1830a81b2faf) | ![Simulator Screenshot - iPhone 14 Pro - 2023-09-18 at 12 33 23](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/68c3346e-d700-4d75-8d12-aa5bce840753)


As you can see in the screenshots, the size of the buttons shrinks as the size of the text inside the buttons shrinks. I think it would be more correct to add `specialButtonStyle`. Or extra styles adjusted for all button sizes. But all styles will be the same. For example:

```
  extraLargeButtonTitle: {
    fontSize: normalize(14),
    fontWeight: '600',
    lineHeight: normalize(18),
  },
  largeButtonTitle: {
    fontSize: normalize(14),
    fontWeight: '600',
    lineHeight: normalize(18),
  },
  mediumButtonTitle: {
    fontSize: normalize(14),
    fontWeight: '600',
    lineHeight: normalize(18),
  },
```

or: 
```
  specialButtonTitle: {
    fontSize: normalize(14),
    fontWeight: '600',
    lineHeight: normalize(18),
  }
```

Reply:
I think the code is as it should be right now, isn't it?

new styles created for title are the same, but I have added new styles according to the props to make it easier to update in the future

https://github.com/smart-village-solutions/smart-village-app-app/commit/1a047b5e20ef37fbd3576cd8d20ff7a8ac819a46